README.md

```diff
@@ -197,32 +193,30 @@ target_link_system_libraries(
 # Package the project
 package_project(
   TARGETS my_header_lib project_options project_warnings
```

Comment:
There's a comment about `PRIVATE` target `project_options` and `project_warnings` should not export.